docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -1102,6 +1102,199 @@ notifications are not part of rooms, and errors for these will show. See the
+- `fetchMore` loads up to 50 more AI chats, and is always safe to call.
+- `isFetchingMore` indicates whether more AI chats are being fetched.
+- `fetchMoreError` returns error statuses resulting from fetching more.
+
+##### Pagination example [#useAiChats-pagination-example]
+
+The following example demonstrates how to use the `fetchMore` function to
+implement a “Load More” button, which fetches additional AI chats when clicked.
+The button is disabled while fetching is in progress.
+
+```tsx
+import { AiChat } from "@liveblocks/react-ui";
+import { useAiChats } from "@liveblocks/react/suspense";
+
+function Inbox() {
+  const { chats, hasFetchedAll, fetchMore, isFetchingMore } = useAiChats();
+
+  return (
+    <div>
+      {chats.map((chat) => (
+        <AiChat key={chat.id} chatId={chat.id} />
+      ))}
+      // +++
+      {hasFetchedAll ? (
+        <div>🎉 You're all caught up!</div>
+      ) : (
+        <button disabled={isFetchingMore} onClick={fetchMore}>
+          Load more
+        </button>
+      )}
+      // +++
+    </div>
+  );
+}
+```
+
+#### Error handling [#useAiChats-error-handling]
+
+Error handling is another important aspect to consider when using the
+`useAiChats` hook. The `error` and `fetchMoreError` fields provide information
+about any errors that occurred during the initial fetch or subsequent fetch
+operations, respectively. You can use these fields to display appropriate error
+messages to the user and implement retry mechanisms if needed.
+
+The following example shows how to display error messages for both initial
+loading errors and errors that occur when fetching more inbox notifications.
+
+```tsx
+import { AiChat } from "@liveblocks/react-ui";
+import { useAiChats } from "@liveblocks/react/suspense";
+
+function Inbox() {
+  const { chats, error, fetchMore, fetchMoreError } = useAiChats();
+
+  // Handle error if the initial load failed.
+  // The `error` field is not returned by the Suspense hook as the error is thrown to nearest ErrorBoundary
+  // +++
+  if (error) {
+    return (
+      <div>
+        <p>Error loading AI chats: {error.message}</p>
+      </div>
+    );
+  }
+  // +++
+
+  return (
+    <div>
+      {chats.map((chat) => (
+        <AiChat key={chat.id} chatId={chat.id} />
+      ))}
+
+      {fetchMoreError && (
+        <div>
+          <p>Error loading more AI chats: {fetchMoreError.message}</p>
+          <button onClick={fetchMore}>Retry</button>
+        </div>
+      )}
+    </div>
+  );
+}
+```
+
+👉 A [Suspense version][] of this hook is also available, which will never
+return a loading state and will throw error to the nearest ErrorBoundary if
+initial load failed.
+
+### useAiChat
+
+Returns information about an AI chat, for example it’s title and metadata.
+
+```tsx
+import { useAiChat } from "@liveblocks/react/suspense";
+
+const { title, metadata, error, isLoading } = useAiChat("my-chat-id");
+```
```

Comment:
I was going to make a note that this hook does not exist yet, but… I just saw @nimeshnayaju did implement this already 🙏 ! (Either I missed it, or I completely forgot about it.)